src/esn.calendar.libs/app/services/fullcalendar/planning-render-event.service.js

```diff
@@ -46,6 +47,30 @@ function calFullCalendarPlanningRenderEventService(
 
         tr.insertBefore(dot, tds[0]);
       }
+
+      function addStyleForEvent(calendar, element, event) {
+        calendar.getOwner().then(function(owner) {
+          const userAsAttendee = calEventUtils.getUserAttendee(event, owner);
+
+          if (userAsAttendee) {
+            switch (userAsAttendee.partstat) {
+            case 'ACCEPTED':
+              element.addClass('planning-event-accepted');
+              break;
+            case 'TENTATIVE':
+              element.addClass('planning-event-tentative');
+              element.find('.fc-list-item-time').prepend('<i class="mdi mdi-help-circle "/>');
+              break;
+            case 'DECLINED':
+              element.addClass('planning-event-declined');
```

Comment:
Just to be safe, please check my PR here https://github.com/OpenPaaS-Suite/esn-frontend-calendar/pull/191 and check if we need to apply the same thing for `planning-event-declined`

Reply:
And also add the test case for that if you need to change it

Reply:
@piavgh It didn't seem to affect planning section